docs/appendices/release-notes/5.4.5.rst

```diff
@@ -65,6 +65,11 @@ Fixes
   ``pg_catalog.pg_namespace`` if a user had privileges on a table within a
   schema, but no privileges on the schema itself.
 
+- Fixed an issue that caused querying on un-indexed columns to return empty
+  results. The expected behaviour was to have an exception thrown but as part
+  of this fix, un-indexed columns as well as columns without doc-values are now
+  queryable.
```

Comment:
Must go to `5.5.1` now.

Reply:
I think we decided to treat this as a feature/breaking change so, `5.6.0`

Reply:
Since we decided, instead of throwing proper exceptions for unindexed columns, to change the behavior and allow to filter on them, Imho it's definitely not a fix, and since it's a change in behavior, I suggested a Breaking change, as before that users would expect an exception if the columns are not index.

@mfussenegger, what's your take?

Reply:
I'd say go with a regular change. This isn't really all that different to adding a new scalar or statement type - if a user relied on an error because a scalar didn't exist, it would also break.  If we start treating these as breaking, then everything becomes breaking and it becomes a useless distinction.

Reply:
Thank you @mfussenegger , I agree.

Reply:
Thank you all, added an entry in `5.6.0` instead. Please have a look.